faq.md

```diff
@@ -29,3 +29,8 @@ following:
 ```
 $ oc new-app --template=oshinko-pyspark-build-dc -p GIT_URI=[your source repo] -e SPARK_OPTIONS='--packages org.apache.spark:spark-sql-kafka-0-10_2.11:2.1.0'
 ```
+
+
+## Why java.io.IOException: failure to login from org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject ?
```

Comment:
i think we should change this to "Why am I getting `java.io.IOException: failure to login from org.apache.hadoop.security.UserGroupInformation.loginUserFromSubject`?"